packages/@okta/i18n/src/properties/login.properties

```diff
@@ -1277,6 +1277,8 @@ oie.user.profile.division = Division
 oie.user.profile.department = Department
 oie.user.profile.managerId = Manager ID
 oie.user.profile.manager = Manager
+oie.selfservice.unlock_user.landing.to.app.success.message = Account successfully unlocked! Verify your account with a security method to continue.
+oie.selfservice.unlock_user.landing.to.app.signing.in.message = Account successfully unlocked!
```

Comment:
These already exist in master, we just have not backported yet, adding now to prevent content issue (server side sends `<br>` tag in text)

See: https://github.com/okta/okta-signin-widget/blob/b00690bacaef60dec1e1508c04993ad1dcea56ec/packages/@okta/i18n/src/properties/login.properties#L1304-L1305

Reply:
@glenfannin-okta If the server side sends <br> tag should this also have the <br> tag then ?

Reply:
As discussed please file a JIRA for the home team to remove the br tag from the enduser-backend bundle to keep this string in sync.

Reply:
🙏 Created the [ticket](https://oktainc.atlassian.net/browse/OKTA-579103), I will follow up with that team as well to make sure it is handled.